docs/root/api-v3/config/watchdog/watchdog.rst

```diff
@@ -1,3 +1,5 @@
+.. _watchdog_overview:
+
```

Comment:
Necessary? BOM injection?

Reply:
What does BOM stand for?

Reply:
Byte order mark, bit I missed that you had injected the .. topic framing.